.travis.yml

```diff
@@ -17,6 +17,14 @@ install:
   - pip3 install -r requirements.txt
   - python3 setup.py --getdeps --log
   - pip3 install . --upgrade
+  - if [[ -z "$TRAVIS_TAG" ]]; then exit 0; fi
```

Comment:
Will it actually exit the script? For example, each Docker's `RUN` command executes in a separate script. Might not be the case with Travis, though.

Reply:
Looks like it does (from this PR CI at least).